app/modules/booking/email-helpers.ts

```diff
@@ -156,44 +156,35 @@ export const checkinReminderEmailContent = ({
     )}.`,
   });
 
-export async function sendCheckinReminder(
+export function sendCheckinReminder(
```

Comment:
Okey so based on this we have to review every place where we call sendEmail and replace `await` with `void`